src/lib/functions.ts

```diff
@@ -6,13 +6,46 @@ import { logWarningMessage } from './logger.js'
 import GleeMessage from './message.js'
 import { GleeFunction } from './index.d'
 import Glee from './glee.js'
-import { gleeMessageToFunctionEvent } from './util.js'
+import { gleeMessageToFunctionEvent, validateData } from './util.js'
 import { pathToFileURL } from 'url'
 
 interface FunctionInfo {
   run: GleeFunction,
 }
 
+const OutboundMessageSchema = {
+  type: 'object',
+  properties: {
+    payload: { type: ['string', 'object'] },
```

Comment:
Payload can be anything so I'd just make it like this:

```suggestion
    payload: {},
```